.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true,
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "overrides": [
+    {
+      "files": ["**/*.test.js", "**/*.cy.js"],
```

Comment:
Instead of merging these blocks, you should have one for cypress and one for jest since they will need different rule sets.